test/test_filters.py

```diff
@@ -51,6 +51,26 @@ def test_(request, case):
     )
 
 
+def test_tag_filter(testdir):
+    # When `method` is specified
+    parameters = {"parameters": [integer(name="id", required=True)]}
+    testdir.make_test(
+        """
+@schema.parametrize(tag="bar")
+@settings(max_examples=1)
+def test_(request, case):
+    request.config.HYPOTHESIS_CASES += 1
+    assert case.path == "/v1/bar"
+    assert case.method == "GET"
+""",
+        paths={"/foo": {"get": parameters.update(tag="foo")}, "/bar": {"post": parameters.update(tag="bar")}},
```

Comment:
`dict.update` returns None and updates the dictionary in place. It should be something like this:

```
{**parameters, "tags": ["foo"]}
```

Note, there should be "tags" key according to the spec - https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#fixed-fields-5
and it should be a list of strings.